src/Microsoft.OData.Client/ISendingRequestEventArgsInterface.cs

```diff
@@ -0,0 +1,14 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ISendingRequestEventArgsInterface.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Client
+{
+    internal interface ISendingRequestEventArgsInterface

```

Comment:
>Interface [](start = 47, length = 9)

Don't generally include the name "Interface" on the interface (the "I" prefix means Interface)